src/frontend/src/lib/components/ui/SlidingInput.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import { Backdrop } from '@dfinity/gix-components';
+	import { onMount } from 'svelte';
+	import { fade, slide } from 'svelte/transition';
+	import { afterNavigate } from '$app/navigation';
+	import IconClose from '$lib/components/icons/lucide/IconClose.svelte';
+	import ButtonIcon from '$lib/components/ui/ButtonIcon.svelte';
+	import InputTextWithAction from '$lib/components/ui/InputTextWithAction.svelte';
+	import { SLIDE_PARAMS } from '$lib/constants/transition.constants.js';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let testIdPrefix: string;
+	export let inputValue: string;
+	export let inputPlaceholder: string | undefined;
+	export let disabled = false;
+	export let ariaLabel: string;
+
+	let visible = false;
+
+	let button: HTMLButtonElement | undefined;
+	let inputElement: HTMLInputElement | undefined;
+
+	const handleOpen = (e?: Event) => {
+		e?.stopPropagation();
+		if (visible) {
```

Comment:
isn't it always defined?

```suggestion

	const handleOpen = (e: Event) => {
		e.stopPropagation();
		if (visible) {
```

Reply:
not when we open it in the onMount. But we dont need the stopPropagation anymore I remove it